tests/functional/adapter/test_python_model.py

```diff
@@ -29,7 +29,7 @@ def models(self):
         return {"30_sec_python_model.py": blocks_for_thirty_sec}
 
     def test_model_times_out(self, project):
-        result, output = run_dbt_and_capture(["run"], expect_pass=False)
+        result, output = run_dbt_and_capture(["run", "--debug"], expect_pass=False)
```

Comment:
I'm pretty sure `"--debug"` needs to be the first item in the list. I ran into issues when I placed it anywhere else because the logs didn't get produced. I'm actually kind of surprised this test passed since you're checking the logs on line 34.

Reply:
This got a left in by accident